WEB-INF/classes/gov/noaa/pfel/erddap/jte/OutOfDateDatasets.java

```diff
@@ -0,0 +1,20 @@
+package gov.noaa.pfel.erddap.jte;
+
+import com.cohort.util.Calendar2;
+import gov.noaa.pfel.erddap.util.EDMessages.Message;
+import gov.noaa.pfel.erddap.util.EDStatic;
+import java.text.MessageFormat;
+
+public class OutOfDateDatasets {
```

Comment:
Since none of the methods in `OutOfDateDatasets` are specific to out-of-date datasets, I think it may be better to move all of these methods to a `TemplateUtil` or `TemplateHelper` class that can be reused in multiple templates.